### PR TITLE
Admin create user

### PR DESCRIPTION
**Pull request by ttuan**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def new
+  end
+
+  def create
+    if @user.save
+      flash[:success] = t "flash.user.create_success"
+      redirect_to new_admin_users_path
+    else
+      flash[:fail] = t "flash.user.create_fail"
+      render "new"
```
Comment (hellovietnam93) [inline app/controllers/admin/users_controller.rb]: using :new instead of

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t "staticpages.framgia"
+    page_title.empty? ? base_title : page_title + " | " + base_title
```
Comment (hellovietnam93) [inline app/helpers/application_helper.rb]: using "#{page_title} | #{base_title}"

File: app/views/admin/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <div class="input-group col-md-12">
+        <span class="glyphicon glyphicon-eye-open"></span>&nbsp;
+        <%= f.label :password_confirmation %>
+        <%= f.password_field :password_confirmation, class: "form-control",
+          placeholder: t("user.place_holder.password_confirmation") %>
+      </div>
+
+      <div class="input-group">
+        <h4><%= t "user.role" %></h4>
+        <%= f.select :role, User.roles.keys.map {|role| [role.titleize,role]} %>
```
Comment (hellovietnam93) [inline app/views/admin/users/new.html.erb]: add space after ,

**Comment by minhld1709** [comment]
OK

**Comment by hellovietnam93** [comment]
OK

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users, path: "auth", path_names: {sign_in: "login", sign_out: "logout"}
+  
+  namespace :admin do
+    resource :users
```
Comment (dieunb) [inline config/routes.rb]: namespace admin

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -1,4 +1,9 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :authenticate_user!
+
+  rescue_from CanCan::AccessDenied do |exception|
+    flash[:alert] = exception.message
+    redirect_to current_user.supervisor? ? root_path : (current_user.admin? ? root_path : new_user_session_path)
```
Comment (dieunb) [inline app/controllers/application_controller.rb]: Refacor nhé

**Pull request merged**